SimPEG/utils/code_utils.py

```diff
@@ -821,7 +821,7 @@ def validate_integer(property_name, var, min_val=-np.inf, max_val=np.inf):
     """
     try:
         var = int(var)
-    except:
+    except TypeError:
```

Comment:
Should likely be `(ValueError, TypeError)`. If the item is a single entity usually a value error is thrown.